src/main/java/com/overops/azure/boards/view/DescriptionGenerator.java

```diff
@@ -0,0 +1,48 @@
+package com.overops.azure.boards.view;
+
+import com.github.jknack.handlebars.Handlebars;
+import com.github.jknack.handlebars.Template;
+import com.github.jknack.handlebars.helper.ConditionalHelpers;
+import com.github.jknack.handlebars.io.ClassPathTemplateLoader;
+import com.github.jknack.handlebars.io.TemplateLoader;
+import com.overops.azure.boards.AzureBoardsException;
+import com.overops.azure.boards.model.EventView;
+
+import java.io.IOException;
+import java.net.URL;
+
+/**
+ * Description Generator
+ *
+ * Generates HTML for the Description field using the Handlebars template engine
+ */
+public class DescriptionGenerator {
+
+    /**
+     * Generates the description HTML from the EventView model
+     * @param event
+     * @return an HTML string
+     */
+    public String generate(EventView event) {
+        try {
+            TemplateLoader loader = new ClassPathTemplateLoader(){
+                @Override
+                protected URL getResource(String location) {
+                    return DescriptionGenerator.class.getResource(location);
+                }
+            };
```

Comment:
Maybe add a comment on why you had to do it this way knowing there was class loading issues.

Reply:
Good call!